app/models/megaphone/episode.rb

```diff
@@ -163,11 +187,8 @@ def check_audio!
       # Re-request the megaphone api
       find_by_megaphone_id
 
-      # get the audio attributes from feeder
-      set_audio_attributes
-
-      # check to see if the audio on mp matches
-      if audio_is_latest?
+      # check to see if the audio on mp matches latest delivery
+      if original_filename == delivery_status(true).source_filename
```

Comment:
Turns out you _can_ use the `original_filename` this way, but it has to be explicitly set - no problem! we'll do that in the code for setting audio attributes below...